akka-cluster/src/main/resources/reference.conf

```diff
@@ -34,6 +34,11 @@ akka {
     # in case of a network partition the persistent actors in the non-surviving partitions
     # must be stopped before corresponding persistent actors are started somewhere else.
     down-removal-margin = 20s
+
+    # Allow new members to join the cluster when there is no convergence.
+    # These new members will have the state 'WeaklyUp' until convergence is reached and leader
+    # can move them to 'Up'. This feature must be off if some members are running Akka 2.3.X.
+    allow-join-new-members-while-no-convergence = off
```

Comment:
Sure, I will change the name of the flag to be more strict about the status and create a pull request to master. Hopefully I can do it by the end of the day today. Thanks for reviewing!